filebeat/scripts/generator/fileset/main.go

```diff
@@ -40,9 +40,9 @@ func generateModule(module, fileset, modulePath, beatsPath string) error {
 	}
 
 	templatesPath := path.Join(beatsPath, "scripts", "module")
-	filesToCopy := []string{path.Join("fields.yml"), path.Join("docs.asciidoc")}
```

Comment:
Both `fields.yml` and `docs.asciidoc` templates are under `scripts/module`. You should not change this path, otherwise templates won't be found.

Reply:
My bad. You seem to be correct here.

Reply:
Yes, in docs https://www.elastic.co/guide/en/beats/devguide/current/filebeat-modules-devguide.html and exists modules it is in _meta everywhere. So, I think generated also must be in _meta. Please confirm that correction right and I don't need to revert it.

Reply:
I tested you script. It's correct.